test/e2e_env/multizone/sync/sync.go

```diff
@@ -187,6 +213,84 @@ spec:
 			}, "30s", "1s").Should(ContainSubstring(name))
 		}
 
+		It("should not sync secret from zone to global", func() {
+			secretName := "global-and-zone-secret"
+			zoneSecret := fmt.Sprintf(`
+apiVersion: v1
+kind: Secret
+metadata:
+  name: %s
+  namespace: %s
+  labels:
+    kuma.io/mesh: %s
+data:
+  value: dGVzdAo=
+type: system.kuma.io/secret `, secretName, Config.KumaNamespace, meshName)
+			globalSecret := fmt.Sprintf(`
+type: Secret
+name: %s
+mesh: %s
+data: Z2xvYmFsCg==`, secretName, meshName)
+			newGlobalSecret := fmt.Sprintf(`
+type: Secret
+name: new-global-secret
+mesh: %s
+data: bmV3Z2xvYmFsCg==`, meshName)
+			Expect(multizone.KubeZone1.Install(YamlK8s(zoneSecret))).To(Succeed())
+			Eventually(func(g Gomega) {
+				out, err := multizone.KubeZone1.GetKumactlOptions().RunKumactlAndGetOutput("get", "secrets", "--mesh", meshName, "-o", "yaml")
+				g.Expect(err).ToNot(HaveOccurred())
+				g.Expect(strings.Count(out, fmt.Sprintf("kuma.io/display-name: %s", secretName))).To(Equal(1))
+			}, "30s", "1s").Should(Succeed())
+			// should not be synced to global
+			Consistently(func(g Gomega) {
+				out, err := multizone.Global.GetKumactlOptions().RunKumactlAndGetOutput("get", "secrets", "--mesh", meshName, "-o", "yaml")
+				g.Expect(err).ToNot(HaveOccurred())
+				g.Expect(strings.Count(out, fmt.Sprintf("kuma.io/display-name: %s", secretName))).To(Equal(0))
+			}, "10s", "1s").Should(Succeed())
+
+			Expect(multizone.Global.Install(YamlUniversal(globalSecret))).To(Succeed())
+			// should not override existing in zone
+			Eventually(func(g Gomega) {
+				out, err := multizone.Global.GetKumactlOptions().RunKumactlAndGetOutput("get", "secrets", "--mesh", meshName, "-o", "yaml")
+				g.Expect(err).ToNot(HaveOccurred())
+				g.Expect(strings.Count(out, secretName)).To(Equal(1))
+				g.Expect(strings.Count(out, "Z2xvYmFsCg==")).To(Equal(1))
+			}, "30s", "1s").Should(Succeed())
+			Consistently(func(g Gomega) {
```

Comment:
Do we need to use `Consistently` here for Zone? That would lead to a flaky error once network latency. Maybe `Eventually` is enough right?